### DISCUSSION TITLE
move asdf-poetry here

### DISCUSSION THREAD

**Discussion by crflynn** [asdf-community/.github#16] [General]
I'd like to move https://github.com/crflynn/asdf-poetry here.

**Comment by smorimoto** [comment]
Cool! Could you invite me as an administrator to the repository for repository transfer? Then I will invite you to this organization.

**Reply by smorimoto** [reply]
Um, seems not have enough privileges...

**Reply by crflynn** [reply]
I'm not sure how to elevate permissions. Can I initiate the transfer myself?

**Reply by smorimoto** [reply]
I just invited you.

**Reply by crflynn** [reply]
It's transferring.

**Reply by crflynn** [reply]
https://github.com/asdf-community/asdf-poetry

**Reply by smorimoto** [reply]
Cool! I will import to Terraform and set up everything.

**Reply by smorimoto** [reply]
Done: https://github.com/asdf-community/infrastructure/commit/8557f2f53ea092faad59c3c77b1a9d57181e825f
